gsy_framework/sim_results/imported_exported_energy.py

```diff
@@ -109,6 +112,12 @@ def _accumulate_imported_exported_energy(self, community_dict: Dict, core_stats:
                     "imported_from_grid",
                     trade["energy"],
                 )
+                # Track community imported energy from grid
+                add_or_create_key(
+                    self.imported_exported_energy[community_key][self._market_slot],
+                    "imported_from_grid",
```

Comment:
If the community is the seller, this should be: "exported_to_grid", right?
And in line 133 it should be "imported_from_grid". Or is my brain not working correctly?

Reply:
No, it is actually the other way around: if the community is the seller name, at least with the current GSY Maps communities, this implies that for both the home and the community the energy is imported from the grid. In this market, the flow of such a trade is `InfiniteBus -> Community -> Home 1`. The export direction is `Home 1 -> Community -> Infinite Bus` in which case the Community is the buyer.

Reply:
Ah yes, this is the Community area agent that is connected to the higher market. I always confuse this. Thanks a lot for explaining!